backend/pharmacy/api/tests/unit/medicial/test_medical_medicine_list.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2022 by YadavGulshan@Github, < https://github.com/YadavGulshan >.
+#
+# This file is part of < https://github.com/Yadavgulshan/PharmaService > project,
+# and is released under the "BSD 3-Clause License Agreement".
+# Please see < https://github.com/YadavGulshan/pharmaService/blob/master/LICENCE >
+#
+# All rights reserved.
+
+from random import random
+
+from rest_framework.test import APITestCase
+
+from pharmacy.api.tests.setup import Service
+
+
+class CheckMedicalMedicine(APITestCase):
+    def setUp(self):
+        self.factory, self.client, self.header = Service.setup_auth_user()
+        res = Service.setupMedicalShop(client=self.client, header=self.header)
+        self.medicalId = res.data["medicalId"]
+        Service.setupMedicineForAShop(
+            client=self.client, header=self.header, medicalId=self.medicalId
+        )
+
+        # Create another user
+        self.factory, self.client, self.header2 = Service.setup_auth_user(
+            username="user2",
+            password="user2",
+            email="someemail@gmail.com",
+            first_name="user",
+            last_name="user",
+        )
```

Comment:
is it a guest user or something else?

Reply:
No it's a normal user.... Shit I wanted to test staffd 🤦